package.json

```diff
@@ -6,18 +6,23 @@
     "build": "fable-splitter -c splitter.config.js --define RELEASE",
     "start": "fable-splitter -c splitter.config.js -w --define DEBUG",
     "compile-for-test": "fable-splitter -c splitter.config.js --define TEST",
-    "cold-start": "fable-splitter -c splitter.config.js --define DEBUG"
+    "cold-start": "fable-splitter -c splitter.config.js --define DEBUG",
+    "reset:packager": "watchman watch-del-all && node_modules/react-native/scripts/packager.sh --reset-cache"
```

Comment:
what's this doing? and how to use it?

Reply:
It is useful to resolve some metro packager errors. For example, sometimes metro gives you red screen with similar instructions when it can't find files after renaming, moving to different folders or reinstalling dependencies.

Reply:
build.cmd runtests